card/lib/generators/card/templates/create_cards_tables.erb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
```

Comment:
This is an odd use of a generator, isn't it?

Also obviously very non-DRY.  Should find a way to use the schema.rb.

As mentioned elsewhere, I don't really like the prefix idea.  It would cause a massive headache.

Really, before you worry too much about automating the setup of an engine deck, I would just experiment with setting one up.

I would expect, for example, that creating a new engine class.

For app decks you have a MyApp::Application class that inherits from Wagn::Application (soon Decko::Application) which in turn inherits from Rails::Application.

I would think that when you create an Engine you would get a MyDeck::Engine class that inherits from Decko::Engine (which inherits from Rails::Engine).  Have you experimented with that?  Do we have a hello world of a deck within a rails app?